cmd/root.go

```diff
@@ -49,8 +49,9 @@ var (
 )
 
 const (
-	pollingInterval  = 7 * time.Second
-	validationWindow = 15
+	pollingInterval      = 7 * time.Second
+	queueRecheckInterval = 5 * time.Minute
```

Comment:
Wouldn't it be nice to make this a CLI argument with a default value? Backwards compatible, but still configurable.

(kinda applies to all our hard coded constants like batch size, validation window, etc, but that's not in scope here)

Reply:
Yeah I dig that idea 👍